prisma/schema.prisma

```diff
@@ -15,17 +15,18 @@ datasource db {
 }
```

Comment:
**praise**: The use of the @relation attribute is well-implemented to establish the relationship between Link and User models.